exporter/elasticsearchexporter/model_test.go

```diff
@@ -890,7 +890,7 @@ type OTelRecord struct {
 	SpanID                 OTelSpanID           `json:"span_id"`
 	Timestamp              time.Time            `json:"@timestamp"`
 	ObservedTimestamp      time.Time            `json:"observed_timestamp"`
-	TraceFlags             uint32               `json:"trace_flags"`
+	TraceFlags             uint32               `json:"-"`
```

Comment:
Is this a leftover that can be removed or does it still serve a purpose?

Reply:
Cleaned it up along with some misses in model_test. Might need to rethink the model_test later, as it doesn't really catch what it should catch.